src/controllers/issue-controller.ts

```diff
@@ -82,6 +83,58 @@ export class IssueController {
         });
     }
 
+    private async quickUpdate() {
+        const memberships = await this.redmine.getMemberships(this.issue.project.id);
+        const possibleStatuses = await this.redmine.getIssueStatusesTyped();
+
+        const statusChoice = await vscode.window.showQuickPick(
+            possibleStatuses.map(status => {
+                return {
+                    "label": status.name,
+                    "description": "",
+                    "detail": "",
+                    "status": status
+                }
+            }),
+            {
+                placeHolder: `Current: ${this.issue.status.name}`
+            }
+        );
+        if(!statusChoice) {
+            return;
+        }
+
+        const desiredStatus = statusChoice.status;
+
+        const assigneeChoice = await vscode.window.showQuickPick(
+            memberships.map(membership => {
+                return {
+                    "label": membership.userName,
+                    "description": "",
+                    "detail": "",
+                    "assignee": membership
+                }
+            }),
+            {
+                placeHolder: `Current: ${this.issue.assigned_to.name}`
+            }
+        );
+        if(!assigneeChoice) {
+            return;
+        }
+
+        const desiredAssignee = assigneeChoice.assignee;
+        const message = await vscode.window.showInputBox({ placeHolder: "Message" });
+
+        const bulkUpdate = new QuickUpdate(this.issue.id, message, desiredAssignee, desiredStatus);
+        const updateResult = await this.redmine.applyQuickUpdate(bulkUpdate);
```

Comment:
There should be an error handler. If `doRequest` in `applyQuickUpdate` throws an error (e.g. not authorized, internal server error, no connection etc.), it won't be catched by re-read mechanism in `applyQuickUpdate`.